test/index.test.js

```diff
@@ -344,4 +344,39 @@ if (ciJob() === 1) {
     expect(stderr).toEqual('')
     expect(stdout).toEqual('esm d\nesm a\nesm b\nesm browser c\nesm lib\n')
   })
+
+  it('copy package.json fields as a conditions for exports field', async () => {
+    let [normalizeCss] = await copyDirs('normalize-css')
+    await processDir(normalizeCss)
+
+    let packageJSON = await readFile(join(normalizeCss, 'package.json'))
+
+    expect(packageJSON.toString()).toMatchInlineSnapshot(`
```

Comment:
Let’s improve code readability by simplifying the test with the direct object compare:

```js
expect(packageJSON).toEqual({
  name: 'normalize-css',
  …
})
```

Reply:
Sure. Just side note this comes with maintenance cost since now there is no way simply `jest -u` to update snapshot.. and jest does not show "smart" diff in this case..

Reply:
Updated